### DISCUSSION TITLE
Does Cake support reading INI files?

### DISCUSSION THREAD

**Discussion by mgnslndh** [cake-build/cake#3258] [Q&A] [answered]
I've tried to search for INI file support but could not find any. Does Cake support reading INI files? I'm asking in case I've missed something.

**Comment by nils-a** [comment, marked as answer]
@mgnslndh I don't think you missed something. There seems to be no support to read ini files currently.

**Comment by pascalberger** [comment]
I don't know of any extensions, but since Cake is just C#, you can use any approach you would use in a console application, like for example the [ini-parser NuGet package](https://www.nuget.org/packages/ini-parser/).

**Reply by mgnslndh** [reply]
Yes, I ended up with a simple Linq statement that worked for my purposes. I was mostly curious if there was something built in I could use. Thanks!
